lib/src/chart/base/base_chart/base_chart_data.dart

```diff
@@ -72,6 +72,74 @@ class FlTouchData {
   const FlTouchData(this.enabled, this.enableNormalTouch);
 }
 
+///***** AxisTitleData *****/
+
+/// This class holds data about the description for each axis of the chart.
+class FlAxisTitleData {
+  final bool show;
+
+  final AxisTitle leftTitle, topTitle, rightTitle, bottomTitle;
+
+  const FlAxisTitleData({
+    this.show = true,
+    this.leftTitle = const AxisTitle(reservedSize: 16),
+    this.topTitle = const AxisTitle(reservedSize: 16),
+    this.rightTitle = const AxisTitle(reservedSize: 16),
+    this.bottomTitle = const AxisTitle(reservedSize: 16),
+  });
+
+  static FlAxisTitleData lerp(FlAxisTitleData a, FlAxisTitleData b, double t) {
+    return FlAxisTitleData(
+      show: a.show || b.show,
```

Comment:
I had the problem that when lerping from `show=true` to `show=false` everything jumped, because the text suddenly disappeared and all of the margins changed. But in hindsight, this is not the right place to use this hack, so I will change it.